docs/infrastructure/administration.md

```diff
@@ -1,215 +1,37 @@
-    app to the local Git repository (to save having to pass `--app` each time) is not helpful.
-    Instead, we recommend adding aliases similar to the following to your bash profile:
-
-```bash
-alias thd='HEROKU_APP=treeherder-prototype heroku'
-alias ths='HEROKU_APP=treeherder-stage heroku'
-alias thp='HEROKU_APP=treeherder-prod heroku'
-```  
-
-This allows commands to be run against a specific app with minimal typing:
-
-```bash
-ths config
-```
+## Common Operations
 
 ### Deploying Treeherder
 
-Deployments occur via Heroku's [GitHub integration] feature, with prototype/stage typically
-set to auto-deploy from the `master` branch, and production from the `production` branch.
-This is controlled via the "deploy" tab in the Heroku app dashboard:
-[prototype][deploy-prototype] | [stage][deploy-stage] | [prod][deploy-prod].
-A comparison of the Git revisions deployed to each environment can be seen using [What's Deployed].
-
-After a push is made to an auto-deployed branch, Heroku will wait for the successful completion of
-the [CircleCI build], before initiating the deployment process.
-The steps described in [deployment lifecycle] then occur, which take about 5 minutes.
-
-Once the deployment is complete, `heroku-bot` will comment in the `#treeherder` IRC channel,
-and for production, an email is sent to the [tools-treeherder] mailing list. Recent deployment
-activity can also be seen on the "activity" tab in the Heroku dashboard for each app.
-
-[github integration]: https://devcenter.heroku.com/articles/github-integration
-[deploy-prototype]: https://dashboard.heroku.com/apps/treeherder-prototype/deploy/github
-[deploy-stage]: https://dashboard.heroku.com/apps/treeherder-stage/deploy/github
-[deploy-prod]: https://dashboard.heroku.com/apps/treeherder-prod/deploy/github
-[what's deployed]: https://whatsdeployed.io/s-dqv
-[circleci build]: https://app.circleci.com/pipelines/github/mozilla/treeherder
-[deployment lifecycle]: architecture.md#deployment-lifecycle
-[tools-treeherder]: https://lists.mozilla.org/listinfo/tools-treeherder
-
-<!-- prettier-ignore -->
-!!! tip
-    To simplify pushing latest `master` to the `production` branch, use this bash alias:
+Stage is set to auto-deploy from the `master` branch, and production from the `production` branch. Prototype deploys from a separate `prototype` branch, but it is not kept in sync with `master` (this will require anyone pushing their code to prototype to reset it to lastest `master` when they're done).
 
-```bash
-# Replace `origin` with the remote name of the upstream Treeherder repository, if different.
-alias deploy='git fetch --all --prune && git push origin remotes/origin/master:production'
-```
+Production deploys are a manual process that is performed by a Treeherder admin roughly twice a week by running a task in Jenkins (under Treeherder -> status -> hover over latest master commit in table and click proceed to initiate the promotion).
```

Comment:
Note to self to review these instructions and add a screen shot once Jeremy grants me access.